src/Arbor/File/Format/Asif/ByteString/Builder.hs

```diff
@@ -10,32 +10,32 @@ module Arbor.File.Format.Asif.ByteString.Builder
   , magicLength
   ) where
 
-import Arbor.File.Format.Asif.Whatever
-import Conduit
-import Control.Lens
-import Control.Monad
-import Data.Bits
-import Data.ByteString.Builder
-import Data.Generics.Product.Any
-import Data.Int
-import Data.Maybe
-import Data.Monoid                     ((<>))
-import Data.String
-import Data.Thyme.Clock
-import Data.Thyme.Clock.POSIX          (POSIXTime, getPOSIXTime)
-import Data.Word
-
-import qualified Arbor.File.Format.Asif.Format as F
-import qualified Arbor.File.Format.Asif.IO     as IO
-import qualified Arbor.File.Format.Asif.Type   as Z
-import qualified Data.ByteString               as BS
-import qualified Data.ByteString.Builder       as B
-import qualified Data.ByteString.Lazy          as LBS
-import qualified Data.ByteString.Lazy.Char8    as LC8
-import qualified Data.Conduit.List             as CL
-import qualified Data.Text.Encoding            as T
-import qualified GHC.IO.Handle                 as IO
-import qualified System.IO.Temp                as IO
+import           Arbor.File.Format.Asif.Whatever
+import           Conduit
+import           Control.Lens
+import           Control.Monad
+import           Data.Bits
+import           Data.ByteString.Builder
+import           Data.Generics.Product.Any
+import           Data.Int
+import           Data.Maybe
+import           Data.Monoid                        ((<>))
+import           Data.String
+import           Data.Thyme.Clock
+import           Data.Thyme.Clock.POSIX             (POSIXTime, getPOSIXTime)
+import           Data.Word
```

Comment:
Is there something different about your IDE set up that it is inserting all these spaces?

Reply:
I have no idea. I haven't changed/configured anything, and the config file seems to be there in the repo...